src/templates/settings/export.html

```diff
@@ -1,4 +1,14 @@
 <div class="settings-section">
+
+  <div class="settings-title settings-section--export">
+    General Export Settings
+  </div>
+  <div class="settings-item">
+    <span class="settings-description" style="display:block"><label for="scaling-factor">Output Scaling Factor</label></span>
+    <span class="settings-description" style="display:block">Currently affects only spritesheet export.</span>
```

Comment:
I know you just applied what you saw a few lines above, but can I ask you to do a bit of extra cleanup ? :) 

Change the spans style=display:block to divs (including the first one in the file you didn't change) and remove the display inline-block from .settings-description rules in settings.css ?

If you move this to the PNG section you can also get rid of the second part of the description.